registry/registry_ui_test.go

```diff
@@ -96,6 +96,8 @@ func TestListNamespaces(t *testing.T) {
 	setupMockRegistryDB(t)
 	defer teardownMockNamespaceDB(t)
 
+	viper.Set("Origin.Port", 8443)
+	viper.Set("Server.WebPort", 8444)
```

Comment:
We're moving toward the randomized ports in our tests, so set these to 0. If you need to create tokens, there should be examples about how to get the appropriate issuer in `handle_http_test.go`

Reply:
Okay, but if my container only forwards specific ports, how can the tests ensure they use the ports my container forwards and has open? This seems like an issue for testing in the local environment.